doc/pages/Install.md

```diff
@@ -60,114 +61,58 @@ sudo install <downloaded file> /usr/local/bin/opam
 
 ## Using your system's package manager
 
-This is generally the recommended way, **when available and up-to-date** (you
-can check [here](Distribution.html) the latest
-available release per distribution). Here is a list of supported distributions:
+This is generally the recommended way, **when available and up-to-date**. You
+can check the following table:
 
-#### Arch Linux
+[![repology package status](https://repology.org/badge/vertical-allrepos/opam.svg?exclude_unsupported=1)](https://repology.org/project/opam/versions)
 
-[![badge](https://repology.org/badge/version-for-repo/arch/opam.svg)](https://repology.org/project/opam/versions)
+If your distribution is up-to-date you can install opam the following way:
```

Comment:
Should this line be before the long table, in case people don't scroll down to see install instructions?

Reply:
done